.github/workflows/release.yml

```
@@ -312,24 +314,24 @@ jobs:
           melos bootstrap
           cd apps/flipper
           flutter clean
-          dart run msix:create -v --install-certificate false
+          dart run msix:create -v --install-certificate false --signtool-options "/td SHA256"
 
       - name: Extract msix_version
         id: get_version
         shell: powershell
         run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)